Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -711,14 +723,13 @@ void DirectoryBlobReader::SetPartitionHeader(DirectoryBlobPartition* partition,
   }
 
   u64 tmd_size;
-  if (wrapped_partition)
+  if (wrapped)
   {
-    const auto& tmd = m_wrapped_volume->GetTMD(*wrapped_partition).GetBytes();
-    auto& new_tmd = m_extra_data.emplace_back(tmd);
+    std::vector<u8> new_tmd = std::move(volume->GetTMD(*wrapped_partition).GetBytes());
```

Comment:
Unnecessary `std::move`. (You already have an rvalue.)

```suggestion
    std::vector<u8> new_tmd = volume->GetTMD(*wrapped_partition).GetBytes();
```